release/goreleaser.opm.Dockerfile

```diff
@@ -2,7 +2,13 @@
 #   build opm images. See the configurations in .goreleaser.yaml
 #   and .github/workflows/release.yaml.
 
-FROM --platform=$BUILDPLATFORM ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.4 as grpc_health_probe
+FROM alpine as grpc_health_probe
```

Comment:
```suggestion
FROM --platform=$TARGETPLATFORM alpine as grpc_health_probe
```

Reply:
it should not be needed, it's the default.